yantra/manager.py

```diff
@@ -81,6 +92,9 @@ def get_plugins(self):
                     # base class
                     if self.plugin_type.base_class in cls.__bases__:
                         self.register_plugin(cls)
+            except Exception as e:
```

Comment:
It was so that the rest of the logic didn't have to be run when an exception happened. But I changed it to only catch the loading and do a continue.